README.md

```diff
@@ -59,9 +63,20 @@ This repository comes with sample data, you can swap out the sample data with yo
 docker run --rm -it -p 8888:8888 -v "/path/to/own/data:/home/jovyan/data" auk-notebook
 ```
 
-> [You must grant the within-container notebook user or group (NB_UID or NB_GID) write access to the host directory (e.g., sudo chown 1000 /some/host/folder/for/work).](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/common.html#docker-options)
+> Note: You must grant the within-container notebook user or group [(NB_UID or NB_GID)](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/common.html#docker-options) write access to the host directory (e.g., sudo chown 1000 /some/host/folder/for/work).
+
+## Types of Visualizations
+
+There are several types of visualizations that you can produce in the Jupyter Notebook. A total of 14 outputs can be generated.
+
+* *Domain Analysis* - provides information about what has been crawled (e.g. which domains) and how often.
```

Comment:
Maybe change the - to a `:` in the list

i.e. 

* *Domain Analysis*: Provides information about what has been crawled (e.g. which domains) and how often.

and same for the other items in this bullet-pointed list.